www/apps/resources/app/recipes/marketplace/examples/restaurant-delivery/page.mdx

```diff
@@ -682,7 +680,7 @@ This returns the list of restaurants in the response.
 
 ### Further Reads
 
-- [What is Remote Query and how to use it](!docs!/advanced-development/modules/remote-query)
+- [What is the Query and how to use it](!docs!/advanced-development/modules/query)
```

Comment:
```suggestion
- [What is Query and how to use it](!docs!/advanced-development/modules/query)
```

Reply:
Would be great to go over it all and ensure it's Query and not the Query. Maybe we can do a search and replace

Reply:
Yes will do, it felt weird just naming it "Query" but let's test it out 😄